cms/bundles/viewsets/bundle.py

```diff
@@ -363,15 +364,40 @@ class BundleIndexView(IndexView):
     """
 
     model = Bundle
+    default_ordering = "name"
 
     def get_base_queryset(self) -> "BundlesQuerySet":
-        """Modifies the Bundle queryset with the related created_by ForeignKey selected to avoid N+1 queries."""
+        """Modifies the Bundle queryset to vary results based on the user capabilities."""
         queryset: BundlesQuerySet = super().get_base_queryset()
 
         if not self.can_manage:
             queryset = queryset.previewable().filter(teams__team__in=self.request.user.active_team_ids).distinct()
 
-        return queryset.select_related("created_by").prefetch_related("teams__team")
+        return queryset
+
+    def filter_queryset(self, queryset: "BundlesQuerySet") -> "BundlesQuerySet":
+        # automatically filter out published bundles if the status filter is not applied
+        if not self.request.GET.get("status"):
+            queryset = queryset.exclude(status=BundleStatus.PUBLISHED)
+
+        return cast("BundlesQuerySet", super().filter_queryset(queryset))
+
+    def order_queryset(self, queryset: "BundlesQuerySet") -> "BundlesQuerySet":
+        if self.ordering in ["status", "-status", "scheduled_publication_date", "-scheduled_publication_date"]:
+            # ensures we have the _updated_at field used by core
+            queryset = self._annotate_queryset_updated_at(queryset)
+
+            match self.ordering:
```

Comment:
These are needed because `scheduled_publication_date` is a property.
And status needs to look at the label, rather than the value